components/shared/filters/modules/AdvancedFilter.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <NeoCollapse
+    :open="expanded"
+    animation="slide"
+    :class="{ 'fluid-padding-left': fluidPadding }">
+    <template #trigger="{ open }">
+      <div
+        class="is-flex is-justify-content-space-between"
+        role="button"
+        :aria-expanded="open">
+        <p class="py-3 px-4 is-size-7 has-text-grey">
+          <span>{{ $t('advancedFilters') }}</span>
+          <span v-if="artView" class="ml-2 has-text-primary"
+            >({{ $t('offer.active') }})</span
+          >
+        </p>
+        <a class="card-header-icon mr-1 has-text-grey">
+          <NeoIcon
+            size="small"
+            :icon="open ? 'chevron-down' : 'chevron-right'" />
+        </a>
+      </div>
+    </template>
+    <div class="px-4">
+      <NeoField>
+        <NeoCheckbox v-model="artView">
+          <span>{{ $t('filters.artView') }}</span>
+          <NeoIcon class="ml-2" size="small" icon="frame" />
+        </NeoCheckbox>
+      </NeoField>
+    </div>
+  </NeoCollapse>
+</template>
+
+<script lang="ts" setup>
+import { useExploreFiltersStore } from '@/stores/exploreFilters'
+import { NeoCheckbox, NeoCollapse, NeoField, NeoIcon } from '@kodadot1/brick'
+
+const exploreFiltersStore = useExploreFiltersStore()
+const route = useRoute()
+const { replaceUrl: replaceURL } = useReplaceUrl()
+
+type DataModel = 'query' | 'store'
+
+const props = withDefaults(
+  defineProps<{
+    expanded?: boolean
+    dataModel?: DataModel
+    fluidPadding?: boolean
+  }>(),
+  {
+    expanded: false,
+    dataModel: 'query',
+    fluidPadding: false,
+  },
+)
+
+const emit = defineEmits(['resetPage'])
+
+const artView =
+  props.dataModel === 'query'
+    ? computed({
+        get: () => route.query?.art_view?.toString() === 'true',
+        set: (value) => applyToUrl({ art_view: String(value) }),
+      })
+    : computed({
+        get: () => exploreFiltersStore.artView,
+        set: (value) => exploreFiltersStore.setArtView(value),
+      })
```

Comment:
This prop `dataModel` has never been passed on the `<AdvancedFilter/>` component.

Reply:
@Jarsen136 fixed, changed to `store` mode on mobile filter.

Previously mobile also worked, except that the filter took effect before the "Apply" button was clicked.